src/client/producer.rs

```diff
@@ -300,6 +300,16 @@ impl BatchProducerBuilder {
     }
 }
 
+/// The result of the produce call.
```

Comment:
```suggestion
/// The result of the [produce](ProducerClient::produce) call.
```

This makes it clearer which API you're referencing here.

Reply:
Updated in [a12992f](https://github.com/influxdata/rskafka/pull/276/commits/a12992f6d3c7da192b0399e3462b89f7188b2758)